system/system_error_monitor/CMakeLists.txt

```diff
@@ -4,11 +4,14 @@ project(system_error_monitor)
 find_package(autoware_cmake REQUIRED)
 autoware_package()
 
-ament_auto_add_executable(${PROJECT_NAME}
-  src/system_error_monitor_node.cpp
+ament_auto_add_library(system_error_monitor_component SHARED
```

Comment:
You can use `ament_auto_add_library(${PROJECT_NAME}_component SHARED` here so that when we prefix packages with `autoware_`, this code won't need to be updated.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/7009/commits/0e5c2df10157b0058d7022911e996757dbc526fd.